lib/Dialect/ESI/cosim/cosim_dpi_server/DpiEntryPoints.cpp

```diff
@@ -0,0 +1,207 @@
+//===- DpiEntryPoints.cpp - ESI cosim DPI calls -----------------*- C++ -*-===//
+//
+// Cosim DPI function implementations. Mostly C-C++ gaskets to the C++
+// RpcServer.
+//
+//===----------------------------------------------------------------------===//
+
+#include "circt/Dialect/ESI/cosim/Server.h"
+#include "circt/Dialect/ESI/cosim/dpi.h"
+
+#include <algorithm>
+#include <cstdlib>
+#include <iostream>
```

Comment:
Please don't include iostream:
https://llvm.org/docs/CodingStandards.html#include-iostream-is-forbidden

Reply:
I went through all the effort of moving the code from "cout << blah" to printf then forgot to delete the include. D'oh!